lib/std/Build/CompileStep.zig

```diff
@@ -1555,7 +1555,9 @@ fn make(step: *Step) !void {
                     try zig_args.append("--test-cmd");
                     try zig_args.append(bin_name);
                     try zig_args.append("--test-cmd");
-                    try zig_args.append("--dir=.");
+                    try zig_args.append("--mapdir=/::.");
+                    try zig_args.append("--test-cmd");
+                    try zig_args.append("--mapdir=/tmp::/tmp");
```

Comment:
I saw some tests failing and using `--dir .` so I assumed I needed to update these flags too

Reply:
Preferably we wouldn't pass `/tmp` here as it means any user who tests their WebAssembly code using std's test step will have this directory appended as a preopen. Unfortunately, it's not yet possible to pass additional arguments in addition to the default. Considering that would require some refactoring, let's keep this as you did here and we can change this in the future instead.

Reply:
I'm a bit uncomfortable with this automatic `/tmp` preopen. Can we perhaps try to solve this in this patch before it lands in master branch?

Reply:
I understand your concerns. The point is ensuring at least two preopens are available, in order for the test to make any sense.

An alternative may be to setup the test env as such:

```
test-root
   + sub1 -> /
   + sub2 -> /tmp
```

then:

```
  --mapdir=/::./test-root/sub1
  --mapdir=/tmp::./test-root/sub2
```

in this case I just need how/where to create these subdirs at a known location before `wasmtime` is started (sorry, real newbie here)